one_big_thing/api_serializers.py

```diff
@@ -140,9 +140,9 @@ def to_representation(self, instance):
         representation = super().to_representation(instance)
 
         for survey_type, survey_keys in SURVEY_FIELDS.items():
-            if survey := instance.surveyresult_set.filter(survey_type=survey_type).last():
-                representation[survey_type] = {
-                    survey_key: survey.data.get(survey_key, "") for survey_key in survey_keys
-                }
+            condensed_survey = dict.fromkeys(survey_keys, "")
+            for survey in instance.surveyresult_set.filter(survey_type=survey_type).all().values():
```

Comment:
```suggestion
            for survey in instance.surveyresult_set.filter(survey_type=survey_type).order_by("-created_at").values():
```
do we need to order to make sure we are overwriting in the right direction?

Reply:
Good catch!

Reply:
[3a9df77](https://github.com/i-dot-ai/one-big-thing/pull/292/commits/3a9df77309c3a748b97c1086d73eed5c015de736)